packages/kbn-monaco/src/esql/lib/ast/definitions/helpers.ts

```diff
@@ -12,12 +12,44 @@ export function getFunctionSignatures(
   { name, signatures }: FunctionDefinition,
   { withTypes }: { withTypes: boolean } = { withTypes: true }
 ) {
-  return signatures.map(({ params, returnType, infiniteParams, examples }) => ({
-    declaration: `${name}(${params.map((arg) => printArguments(arg, withTypes)).join(', ')}${
-      infiniteParams ? ` ,[... ${params.map((arg) => printArguments(arg, withTypes))}]` : ''
-    })${withTypes ? `: ${returnType}` : ''}`,
-    examples,
-  }));
+  return signatures.map(({ params, returnType, infiniteParams, minParams, examples }) => {
+    // for functions with a minimum number of args, repeat the last arg multiple times
```

Comment:
Can you give me an example of a function with a minimum number of args? I am not sure I understand in which function you are referring to so I can't test 😄

Reply:
Functions with a minimum number of args are:
* `case`
* `concat`
* `cidr_match`

But I will extract the fix logic for these and move it to another PR to avoid confusion.